### ISSUE TITLE
How it works / How to enable VoLTE manually

### ISSUE THREAD

**Issue by czl92783719** [kyujin-cho/pixel-volte-patch#324]
**Describe the bug**
How the Pixel IMS app works? How to enable VoLTE myself manually? I don't want to use the goddamn fucking proprietary Shizuku app.

**Comment by czl92783719** [comment]
Pixel 6 GrapheneOS with no root

**Comment by hoangvietfreelancer** [comment]
Shizuku is proprietary? https://github.com/RikkaApps/Shizuku

**Comment by czl92783719** [comment]
> Shizuku is proprietary? https://github.com/RikkaApps/Shizuku
Yes, it forbids the redistribution of self complied or modified versions.

License

The code for this project is available under the Apache-2.0 license.
Exceptions

You are FORBIDDEN to use image files listed below in any way (unless for displaying Shizuku itself).

manager/src/main/res/mipmap-hdpi/ic_launcher.png
manager/src/main/res/mipmap-hdpi/ic_launcher_background.png
manager/src/main/res/mipmap-hdpi/ic_launcher_foreground.png
manager/src/main/res/mipmap-xhdpi/ic_launcher.png
manager/src/main/res/mipmap-xhdpi/ic_launcher_background.png
manager/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png
manager/src/main/res/mipmap-xxhdpi/ic_launcher.png
manager/src/main/res/mipmap-xxhdpi/ic_launcher_background.png
manager/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png
manager/src/main/res/mipmap-xxxhdpi/ic_launcher.png
manager/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png
manager/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png

For the project as a whole, it is not free. You are FORBIDDEN to distribute the apk compiled by you (including modified, e.g., rename app name "Shizuku" to something else) to any store (IBNLT Google Play Store, F-Droid, Amazon Appstore etc.).

**Comment by jerielbenavides** [comment]
Those are the icons/"branding". Can't you put your own and distribute it?

This stops bad actors from keeping malicious forks up on those stores
